API/quickstart.rb

```diff
@@ -0,0 +1,58 @@
+require 'rubygems'
```

Comment:
please, rename this file, since it's difficult to understand what it is responsible for.